src/modules/IscsiClientLib.rb

```diff
@@ -1499,22 +1499,24 @@ def CallConfigScript
     def GetDiscoveryCmd(ip, port, use_fw: false, only_new: false)
       Builtins.y2milestone("GetDiscoveryCmd ip:%1 port:%2 fw:%3 only new:%4",
         ip, port, use_fw, only_new)
-      command = "-m discovery -P 1"
+      command = ["/usr/sbin/iscsiadm", "-m", "discovery", "-P", "1"]
```

Comment:
this changes the return type from String to 
```
# @return [Array<String>]
```

also below, IMHO delete this instead of fixing it
```rb
 publish :function => :GetDiscoveryCmd, :type => "string (string, string, map)"
```

Reply:
good catch. I check all usage, but forgot about that publish